### PR TITLE
tpm_device: check swtpm.pid file after vm destroy

### PR DESCRIPTION
**Pull request by yanqzhan**
swtpm.pid file should not exist after vm destroy, or managedsaved.

Signed-off-by: Yanqiu Zhang <yanqzhan@redhat.com>

### CODE REVIEW

**Comment by yanqzhan** [comment]
To implement bug fix of https://bugzilla.redhat.com/show_bug.cgi?id=2111301.

File: libvirt/tests/src/virtual_device/tpm_device.py [hunk 1]
```diff
@@ -719,6 +720,8 @@ def save_modify_pcrbank(vm_name, active_pcr_banks, pcrbank_change):
                     virsh.snapshot_create_as(vm_name, "sp1 --memspec file=/tmp/testvm_sp1", **virsh_dargs)
                 elif vm_operate in ["restart", "create"]:
                     vm.destroy()
+                    if glob.glob("/run/libvirt/qemu/swtpm/*-%s-swtpm.pid" % vm_name):
```
Comment (chunfuwen) [inline libvirt/tests/src/virtual_device/tpm_device.py:723]: line 723 and 724 are used three times, maybe wrap them into one simple method instead.

Comment (yanqzhan) [reply libvirt/tests/src/virtual_device/tpm_device.py]: Done. Thanks! 
Pls review again.

**Comment by yanqzhan** [comment]
(1/2) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.plain.pcrbank_default.restart_vm.domrename_operate: PASS (101.00 s)
 (2/2) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.normal_test.tpm-crb_model.emulator.plain.pcrbank_default.restore_vm: PASS (76.10 s)

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by cliping — APPROVED** [review]
LGTM

**Pull request merged**